src/sass/app/_common/form/input-field.scss

```diff
@@ -13,6 +13,13 @@
         text-align: center;
         width: 100%;
     }
+    & .inline-icon {
+        position: absolute;
+        top: 8px;
+        left: 8px;
+        z-index: 99;
```

Comment:
can we make the calendar `z-index: 99` with transparent background, and icon `z-index: 98`

Reply:
Why should we do that? 🤔